src/gl.rs

```diff
@@ -68,6 +68,7 @@ pub fn read_pixels(x: GLint, y: GLint, width: GLsizei, height: GLsizei, format:
         ffi::BGRA => 4,
 
         ffi::ALPHA => 1,
+#[cfg(target_os="android")]
```

Comment:
Yep - luminance was deprecated and removed from gl3.3 which is what the desktop bindings now use.